ui/brief.xhtml

```diff
@@ -76,6 +76,7 @@
                     <img class="icon"/>
                     <label class="title"/>
                     <label class="unread-count"/>
+                    <label class="error-status">●</label>
```

Comment:
The main problem is that a brown circle (okay, even a red circle) is not exactly obvious and it lacks any kind of description. How is the user supposed to know it means "this feed failed to load last time"? This definitely needs a textual comment of some kind.

Reply:
I agree. Tooltip added.